site/src/components/pages/hub/block-data-container.tsx

```diff
@@ -213,7 +217,7 @@ export const BlockDataContainer: FunctionComponent<BlockDataContainerProps> = ({
               sx={{
                 overflow: "auto",
                 padding: theme.spacing(4, 4),
-                height: "100%",
+                height: "calc(100% - 40px)",
```

Comment:
Couldn't get a straightforward fix since we need the block preview box and block properties editor to have the same height. Removed the need for `calc` here => [376a584](https://github.com/blockprotocol/blockprotocol/pull/489/commits/376a584c36a57ede16008362e65cdca83f8c1d37)